structuredheadings/plugin.js

```diff
@@ -167,6 +177,57 @@
 
     editor.config.autonumberCurrentStyle =
     editor.config.autonumberCurrentStyle || "1.1.1.1.1.";
+
+    // maps each potential part of a heading preset to
+    // the PolicyStat CSS class for lists
+
+    editor.config.listClassMappings = editor.config.listClassMappings || {
+      //eslint-disable-next-line new-cap
+      "A": new CKEDITOR.style(
+        {
+          name: "List: A. B. C.",
+          element: "ol",
+          attributes: {"class": "list-upper-alpha"}
+        }
+      ),
+      //eslint-disable-next-line new-cap
+      "a": new CKEDITOR.style(
+        {
+          name: "List: a. b. c.",
+          element: "ol",
+          attributes: {"class": "list-lower-alpha"}
+        }
+      ),
+      //eslint-disable-next-line new-cap
+      "1": new CKEDITOR.style(
+        {
+          name: "List: 1. 2. 3.",
+          element: "ol",
+          attributes: {"class": "list-decimal"}
+        }
+      ),
+      //eslint-disable-next-line new-cap
+      "I": new CKEDITOR.style(
+        {
+          name: "List: I. II. III.",
+          element: "ol",
+          attributes: {"class": "list-upper-roman"}
+        }
+      ),
+      //eslint-disable-next-line new-cap
+      "i": new CKEDITOR.style(
+        {
+          name: "List: i. ii. iii.",
+          element: "ol",
+          attributes: {"class": "list-lower-roman"}
+        }
+      ),
+      //eslint-disable-next-line new-cap
+      "clear": new CKEDITOR.style({
```

Comment:
This one has `({` and `})` and the rest split them on two lines each. I like this one better, but regardless, I feel like they should all be the same.